apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/charSequenceContainsCreators.kt

```diff
@@ -277,5 +361,12 @@ infix fun <T : CharSequence> CharSequenceContains.Builder<T, IgnoringCaseSearchB
  *
  * @return The plant to support a fluent API.
  */
+@Deprecated(
```

Comment:
this one would not have been necessary as it is private. No need to revert though, fine as well